examples/web-react/package.json

```diff
@@ -24,9 +25,6 @@
     "@babel/core": "^7.15.0",
     "@babel/preset-env": "^7.15.0",
     "@babel/preset-react": "^7.14.5",
-    "babel-loader": "^8.2.2",
-    "mkdirp": "^1.0.4",
-    "webpack": "^5.50.0",
```

Comment:
`webpack` and `webpack-cli` need to stay here as a reference, this is an example project.

Reply:
They are back again.